crates/rpc/src/types/errors.rs

```diff
@@ -1,32 +1,46 @@
+use actix_web::{HttpResponse, ResponseError, http::StatusCode};
+use derive_more::derive::Display;
```

Comment:
could you remove the `derive_more` crate and just use `thiserror` like we were doing before